support/index.html

```diff
@@ -125,6 +125,24 @@ <h2>Experienced Providers</h2>
           </p>
         </div>
     </div>
+
+    <div class="row tbl">
+        <div class="col-xs-2 cell text-center">
+           <img src="img/terrestris-logo.png"></img>
+        </div>
+        <div class="col-xs-10">
+           <a href="http://www.terrestris.de/en/">terrestris</a> provides
+           top-notch Free and OpenSource GIS services, solutions and training
+           from Bonn, Germany. We offer custom project solutions, connect you
```

Comment:
Minor feedback, since it is the geoserver website doing the talking the description may be more consistent if it uses "terrestris" and "they" (rather than "we").

> terrestris provides top-notch Free and OpenSource GIS services, solutions and training
> from Bonn, Germany. _Offering_ custom project solutions, connectin you
> with communities and provide general support for many FOSS packages
> and libraries. _They_ help customers in the planning, implementation
> and  maintenance of their spatial data infrastructures (SDI) with
> FOSS. GeoServer is often one crucial pillar in such SDIs; please <a href="mailto:info@terrestris.de" title="Send terrestris an email">reach out</a>,
> to see if _terrestris_ can help you to get the best out of FOSS for
> geospatial.